libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -162,6 +162,14 @@ const AP_Param::GroupInfo AP_MotorsMulticopter::var_info[] = {
     // @Increment: 0.1
     // @User: Advanced
     AP_GROUPINFO("SPOOL_TIME",   36, AP_MotorsMulticopter,  _spool_up_time, AP_MOTORS_SPOOL_UP_TIME_DEFAULT),
+
+    // @Param: BOOST_SCALE
+    // @DisplayName: Motor boost scale
+    // @Description: This is a scaling factor for vehicles with a vertical booster motor used for extra lift. It is used with electric multicopters that have an interal combusion booster motor for longter endurance. The output to tke BoostThrottle servo function is set to the current motor thottle times this scaling factor. A higher scaling factor will put more of the load on the booster motor. A value of 1 will set the BoostThrottle equal to the main throttle.
```

Comment:
The output to **tke** BoostThrottle -> The output to **the** BoostThrottle

Reply:
https://youtu.be/6c5m10Wtuow

Hello 
How do you connect the  PWM to the pixhawk to the servo that connects to the Gad Combustion Engine ?

https://youtu.be/6c5m10Wtuow